python/GafferImageUI/ConstantUI.py

```diff
@@ -66,6 +66,16 @@
 
 		],
 
+		"layer" : [
+
+			"description",
+			"""
+			The layer to generate. The output channels will
+			be named ( layer.R, layer.G, layer.B and layer.A ).
```

Comment:
The only way they could see 'em in the viewer right now would be to use a Shuffle to manually copy 'em over to the main RGBA channels. I can hide it if you like, but since the viewer support is nearly at the top of the todo list I don't think it'll be too long before that's irrelevant - with a bit of luck in time for the next release even.